scanpy/tools/_umap.py

```diff
@@ -89,6 +90,8 @@ def umap(
         `spread`.
     copy : `bool` (default: `False`)
         Return a copy instead of writing to adata.
+    method : {{'umap', 'rapids'}}  (default: `'umap'`)
```

Comment:
This function doesn’t use `@doc_params`, so normal braces should be sufficient.

```suggestion
    method : {`'umap'`, `'rapids'`}  (default: `'umap'`)
```